.github/workflows/publish.yaml

```diff
@@ -0,0 +1,32 @@
+name: Deploy to maven central
+on:
+  push:
+    tags:
+      - '\d+.\d+.\d+-SNAPSHOT'
+      - '\d+.\d+.\d+'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        include:
+          - os: ubuntu-latest
+            java-version: 11
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Clone
+        uses: actions/checkout@v2
+      - name: Build
+        uses: eskatos/gradle-command-action@v1
+        with:
+          arguments: build
+      - name: Publish to maven central
+        uses: eskatos/gradle-command-action@v1
+        with:
+          arguments: publish
+        env:
+          GODOT_KOTLIN_GPG_PRIVATE_KEY_ASCII: ${{ secrets.GODOT_KOTLIN_GPG_PRIVATE_KEY_ASCII }}
+          GODOT_KOTLIN_GPG_KEY_PASSPHRASE: ${{ secrets.GODOT_KOTLIN_GPG_KEY_PASSPHRASE }}
+          GODOT_KOTLIN_MAVEN_CENTRAL_TOKEN_USERNAME: ${{ secrets.GODOT_KOTLIN_MAVEN_CENTRAL_TOKEN_USERNAME }}
+          GODOT_KOTLIN_MAVEN_CENTRAL_TOKEN_PASSWORD: ${{ secrets.GODOT_KOTLIN_MAVEN_CENTRAL_TOKEN_PASSWORD }}
```

Comment:
Do you really want to use the same key?

Reply:
I think it can become really painful to manage when keys will be out of date. This key can become the one for utopia rise open source projects. Anyway, it is easy to change if we want to.